File: 2.0/python/webhooks/getWebhookResource.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.webhooks.getWebhookResource(
    resource='CallBroadcast',
    # return only specific fields
    fields='resource,events'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/webhooks/getWebhookResource.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->getWebhookResource();
        $request->getOperationConfig()->setPathParameters(array("resource" => "CALL_BROADCAST"));
        $request->getOperationConfig()->setQueryParameters(array("fields" => "resource,events"));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
